packages/p2p-v2/src/components/AdvertiserName/AdvertiserNameBadges.tsx

```diff
@@ -11,11 +10,9 @@ import './AdvertiserNameBadges.scss';
  * Use cases are usually in My Profile page and Advertiser page used under the advertiser's name
  */
 const AdvertiserNameBadges = () => {
-    const { data: advertiserStats, isLoading } = useAdvertiserStats();
+    const { data: advertiserStats } = useAdvertiserStats();
 
-    if (isLoading || !advertiserStats) return <Loader />;
-
-    const { isAddressVerified, isIdentityVerified, totalOrders } = advertiserStats;
+    const { isAddressVerified, isIdentityVerified, totalOrders } = advertiserStats || {};
```

Comment:
Why not use the loader thou?

Reply:
cause currently in p2p-v1 behaviour, we don't load everything individually, and once we decide on the skeleton loader design, then we can do something for it, also the Loader component also looks weird on each individual part of the myprofile page 😅